test/functional/testCases.js

```diff
@@ -1494,6 +1494,8 @@ const testCases = [
                                 type: 'command'
                             }
                         ],
+                        endpoint: 'http://myendpoint.com',
```

Comment:
Should specific tests (e.g CRUD on the new fields added to Group API) be added?

Reply:
These functional tests has no cover command executions neither registration of commands

Reply:
update group api tests in https://github.com/telefonicaid/iotagent-node-lib/pull/1543/commits/90e5efc9569c4d7423ba33580bbc455abf8e9f85